packages/app/src/lib/client/index.ts

```diff
@@ -719,6 +720,57 @@ export class SunriseStakeClient {
     };
   }
 
+  public async createMetadata(
```

Comment:
We should put all this in the script only as it will only ever be called once and won't be used in the client itself.